pycroft/model/alembic/versions/bc0e0dd480d4_add_bankaccount_owner.py

```diff
@@ -0,0 +1,84 @@
+"""add bankaccount owner
+
+Revision ID: bc0e0dd480d4
+Revises: 55e9f0d9b5f4
+Create Date: 2024-03-16 08:42:48.684471
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "bc0e0dd480d4"
+down_revision = "55e9f0d9b5f4"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("bank_account", sa.Column("owner", sa.String(length=255), nullable=True))
+    op.drop_index("bank_account_activity_imported_at", table_name="bank_account_activity")
+    op.alter_column("log_entry", "author_id", existing_type=sa.INTEGER(), nullable=False)
+    op.alter_column(
+        "membership_fee", "booking_begin", existing_type=postgresql.INTERVAL(), nullable=False
+    )
+    op.alter_column(
+        "membership_fee", "booking_end", existing_type=postgresql.INTERVAL(), nullable=False
+    )
+    op.alter_column("pre_member", "passwd_hash", existing_type=sa.VARCHAR(), nullable=True)
+    op.alter_column("pre_member", "birthdate", existing_type=sa.DATE(), nullable=True)
+    op.alter_column(
+        "room_history_entry", "active_during", existing_type=postgresql.TSTZRANGE(), nullable=False
+    )
+    op.create_index(
+        op.f("ix_room_history_entry_room_id"), "room_history_entry", ["room_id"], unique=False
+    )
+    op.create_index(
+        op.f("ix_room_history_entry_user_id"), "room_history_entry", ["user_id"], unique=False
+    )
+    op.drop_constraint("room_history_entry_room_id_fkey", "room_history_entry", type_="foreignkey")
+    op.drop_constraint("room_history_entry_user_id_fkey", "room_history_entry", type_="foreignkey")
+    op.create_foreign_key(
+        None, "room_history_entry", "room", ["room_id"], ["id"], ondelete="CASCADE"
+    )
+    op.create_foreign_key(
+        None, "room_history_entry", "user", ["user_id"], ["id"], ondelete="CASCADE"
+    )
+    op.alter_column("task", "task_type", existing_type=sa.VARCHAR(length=50), nullable=False)
+    op.alter_column("traffic_volume", "user_id", existing_type=sa.INTEGER(), nullable=False)
+    # ### end Alembic commands ###
```

Comment:
The only statement which should be there is the first one (adding `owner`). Do we have such drastic divergences between theoretical `model` and the actual model in the DB?

Reply:
> The only statement which should be there is the first one (adding owner).

Removed the other ones.

> Do we have such drastic divergences between theoretical model and the actual model in the DB?

I don't know.  I don't know how the generation of this worked exactly either.